New-MtrWallpaper.ps1

```diff
@@ -1,353 +1,353 @@
-         if you have more then one MTR or want to do it remotely!
-	
-         .PARAMETER Check
-         Check if a given Wallpaperfile has the correct resolution
-	
-         .PARAMETER RedComponent
-         RedComponent for the template
-         Default is 100
-	
-         .PARAMETER GreenComponent
-         GreenComponent for the template
-         Default is 100
-	
-         .PARAMETER BlueComponent
-         BlueComponent for the template
-         Default is 100
-	
-         .EXAMPLE
-         New-MtrWallpaper
-		
-         Picks a random image, copies to the root folder, and writes XML file
-	
-         .NOTES
-         None for now. May play with configuring a list of dates and corresponding wallpaper images (think holiday wallpapers).
-		
-         All Images must be exactly 3840×1080, for Single or dual screen!
-		
-         If you copy the wallpaper image (wallpaper.jpg), with or without the settings XML,
-         to different MTR systems the change will not be activated!
-         The MTR doesn't do a reload or check if a file is changed.
-         The MTR will reboot every day at 2:30 (AM), then the changes will be activated.
-		
-         The MTR support .jpg, .jpeg, .png, and .bmp for wallpapers!
-		
-         My version of the script has no enhancements, at least not yet!
-         It's just a minor changed version based on the great work of Pat Richard.
-	
-         .LINK
-         https://www.ucunleashed.com/4323
-	
-         .LINK
-         https://github.com/patrichard/New-MtrWallpaper
-	
-         .LINK
-         https://docs.microsoft.com/en-us/MicrosoftTeams/downloads/ThemingTemplateMicrosoftTeamsRooms_v2.1.psd
-	
-         .LINK
-         https://www.ucit.blog/post/configuring-custom-themes-for-microsoft-teams-skype-room-systems
-	
-         .LINK
-         https://www.bing.com/images/search?q=wallpaper+3840x1080&qpvt=wallpaper+3840x1080&form=IGRE&first=1&cw=1680&ch=939
-   #>
-	
-   [CmdletBinding(ConfirmImpact = 'None',
-   SupportsShouldProcess)]
-   param
-   (
-      [Parameter(ValueFromPipeline,
-      ValueFromPipelineByPropertyName)]
-      [ValidateNotNullOrEmpty()]
-      [Alias('ThisRoot')]
-      [string]
-      $Path = "$env:HOMEDRIVE\Users\Skype\AppData\Local\Packages\Microsoft.SkypeRoomSystem_8wekyb3d8bbwe\LocalState",
-      [Parameter(ValueFromPipeline,
-      ValueFromPipelineByPropertyName)]
-      [switch]
-      $Check,
-      [Parameter(ValueFromPipeline,
-      ValueFromPipelineByPropertyName)]
-      [ValidateRange(0, 100)]
-      [Alias('Red')]
-      [int]
-      $RedComponent = 100,
-      [Parameter(ValueFromPipeline,
-      ValueFromPipelineByPropertyName)]
-      [ValidateRange(0, 100)]
-      [Alias('Green')]
-      [int]
-      $GreenComponent = 100,
-      [Parameter(ValueFromPipeline,
-      ValueFromPipelineByPropertyName)]
-      [ValidateRange(0, 100)]
-      [Alias('Blue')]
-      [int]
-      $BlueComponent = 100
-   )

```

Comment:
These were all hard coded at the original max (which was 100). I've not seen these change anything.